batocera-linux/batocera.linux
# fixes slow citra

- bump to latest nightly
- use the nighly git repo now
- fix citra running slow on x86_64 systems
- hide the mouse pointer when inactive
- remove gui prompt at start
- ensure no telemetry data is sent
